app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,5 @@
+<% provide(:title, "About") %>
+<h1><%= t "About" %></h1>
```

Comment:
I18n + tên biến ko bắt đầu với từ viết hoa, self check toàn bộ html, chị ko comment nữa nhé